packages/block-library/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Breaking Change
```

Comment:
I think this isn't necessarily breaking anything :thinking:

```suggestion
### Enhancement
```

Reply:
Yep, I was clueless. It breaks the styling but it doesn't break any JS API though.